### DISCUSSION TITLE
Welcome to python-interview-questions Discussions!

### DISCUSSION THREAD

**Discussion by learning-zone** [learning-zone/python-basics#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ranjankumarsinghee** [comment]
my name is ranjan kumar singh sir i want intrested in python end to end project indusry based so any project you have or any idea about plz sir help me.

**Comment by learning-zone** [comment]
Hi @ranjankumarsinghee ,

I would suggest to start with small projects first to get better hands-on.  For instance, ToDo List.
Later on you can get lot of project ideas based on your field interest from github.

**Comment by ranjankumarsinghee** [comment]
sir give you your personal gmail id because i share some personal
issues regarding job. like i have huge gap after B.tech and i tell
everything so plz sir send a personal gmail id at my gmail id
***@***.***

**Comment by kushnertodd** [comment]
This is an Introduction to Python Jupyter notebook to teach Python with little or no prior experience in programming.

http://www.toddkushnerllc.com/intro_to_python.html

**Comment by nathvansh** [comment]
[ python coding questions](https://web.talenttitan.com/candidates/interview-preparation/python) for beginners which you can try 

1. Write a program to find the factorial of a number.
2. How do you check if a given year is a leap year?
3. Reverse a string without using built-in methods.

**Comment by WangilaWakhongola** [comment]
What is the difference between is and == in Python? 
supposed asked in an interview what is the right response
